### ISSUE TITLE
 questions about the noise of IMU data and image data

### ISSUE THREAD

**Issue by ghost** [daniilidis-group/mvsec#8]
Hi，First of all, thank you for sharing your dataset.
I have some questions about the noise of IMU data.
First：I want to use the dataset to run VINS mono，I want to know the paramaters as follows:
(1)accelerometer measurement noise standard deviation,
(2)gyroscope measurement noise standard deviation,
(3)accelerometer bias random work noise standard deviation
(4)gyroscope bias random work noise standard deviation
and gravity magnitude in your dataset.
Second: I found there is no image data in rosbag file, how can I get the image data?
Thanks for your time.

**Comment by alexzzhu** [comment]
Hi there,

The IMU on the DAVIS is the MPU 6150, and you can find the specs sheet here: https://www.cdiweb.com/products/detail/mpu6150-invensense/470090/. This should have the statistics you are looking for. Regarding the image data, which bag are you referring to specifically? It should be available for most of them, although the right camera images are missing in outdoor_day1 and 2.

**Comment by ghost** [comment]
@alexzzhu Hi,alexzzhu,  I would like to know the calibration data of IMU when obtaining the data set, well,  Is the device calibrated before data collection?
And I am looking for the Indoor flying (Note: No VI-Sensor data is available for this scene)dataset's image file.
Thanks a lot!

**Comment by alexzzhu** [comment]
Oh I see, no we do not have IMU calibrations beyond geometric extrinsics. The images should be stored in the bag file under /davis/image_raw, if that's what you're using. They're also available in the hdf5 files, I believe under the same name.

**Comment by ghost** [comment]
Please excuse me，calibration is very important for the stability of SLAM/VIO system , I want to know why do not have IMU calibrations beyond geometric extrinsics？

**Comment by alexzzhu** [comment]
I agree that having the IMU parameters would help with state estimation. However, I'm not sure what to tell you aside from the fact that these were not collected.

**Comment by ghost** [comment]
I want to use this dataset to run the VINS-mono system,I need the (1)accelerometer measurement noise standard deviation,
(2)gyroscope measurement noise standard deviation,
(3)accelerometer bias random work noise standard deviation
(4)gyroscope bias random work noise standard deviation
and gravity magnitude in your dataset.
These parameters are very important for the initialization and normal operation of the system, If these parameters are not accurate, I don't know where the IMU can fly?

**Comment by alexzzhu** [comment]
Have you tried running the system with some default set of calibrations? As far as I'm aware these systems shouldn't be so sensitive as to break completely without calibrated values.

**Comment by yao-hexiong** [comment]
> Hi，First of all, thank you for sharing your dataset. I have some questions about the noise of IMU data. First：I want to use the dataset to run VINS mono，I want to know the paramaters as follows: (1)accelerometer measurement noise standard deviation, (2)gyroscope measurement noise standard deviation, (3)accelerometer bias random work noise standard deviation (4)gyroscope bias random work noise standard deviation and gravity magnitude in your dataset. Second: I found there is no image data in rosbag file, how can I get the image data? Thanks for your time.

I failed to run the indoor flying dataset on vinsmono, how is your yaml file set up?
